docroot/themes/custom/yukonca_glider/templates/content/node--site-wide-alert.html.twig

```diff
@@ -78,16 +78,23 @@
   ]
 %}
 {{ attach_library('classy/node') }}
+
 <article{{ attributes.addClass(classes) }}>
 
   {{ title_prefix }}
-  {% if label and not page %}
-    {{ label }}
+  {% if label and not page and not view_mode == 'teaser' %}
+    <h2{{ title_attributes }}>
+      <a href="{{ url }}" rel="bookmark">{{ label }}</a>
+    </h2>
   {% endif %}
   {{ title_suffix }}
+{% set icon %}
+  <i class="fa fa-exclamation-triangle text-[38px]"></i>
```

Comment:
Can you confirm that 38px font size is a value that is not re-used elsewhere?

And can we use rem value instead of px?

Reply:
Confirm it uses only for icon - twice on site. I think it not nessesary to create a new variable.